draft-ietf-quic-transport.md

```diff
@@ -633,7 +632,7 @@ on the two endpoints agreeing on a version.
 A QUIC connection begins with a client sending a handshake packet. Until the
 cryptographic handshake ({{handshake}}) produces 1-RTT keys, packets sent by a
 client MUST include the version in the packet header.  This allows the server to
-identify early packets and to enable version negotiation.
+identify 0-RTT packets and to enable version negotiation.
```

Comment:
I guess that we still disagree about the necessity for marking 0-RTT packets then.  I will remove the sentence in the interests of not blocking the entire PR.  Expect this to come up again.